modAionImpl/src/org/aion/zero/impl/AionBlockchainImpl.java

```diff
@@ -1342,6 +1342,21 @@ private AionBlockSummary applyBlock(IAionBlock block) {
             for (AionTxExecSummary summary : executionSummaries) {
                 receipts.add(summary.getReceipt());
                 summaries.add(summary);
+
+                // save contract creation data to contract details
+                AionTxReceipt receipt = summary.getReceipt();
+                AionTransaction tx = receipt.getTransaction();
+                if (tx.isContractCreationTransaction() && receipt.isSuccessful()) {
+                    track.saveVmType(
+                            tx.getContractAddress(),
+                            TransactionTypeRule.isValidAVMContractDeployment(tx.getTargetVM())
+                                    ? TransactionTypes.AVM_CREATE_CODE
+                                    // FVM contracts do not always have the correct type
+                                    : TransactionTypes.FVM_CREATE_CODE);
+                    if (tx.getTargetVM() != 1) {
+                        System.out.println("Contract creation transaction receipt: " + receipt);
```

Comment:
use log instead of the std output?

Reply:
sorry, this commit was not meant to be added to the PR; I removed it